torch/onnx/_internal/exporter/_ir_passes.py

```diff
@@ -35,9 +35,8 @@ def add_torchlib_common_imports(model: ir.Model) -> None:
 
     try:
         # TODO(justinchuby): Remove this hack and improved onnxscript
-        from onnxscript.function_libs.torch_lib.ops import common as common_ops
```

Comment:
Do we expect it to fail?

Reply:
Actually not. I will remove the try block

Reply:
Done